gcc/multiple_target.cc

```diff
@@ -298,6 +298,20 @@ create_target_clone (cgraph_node *node, bool definition, char *name,
   return new_node;
 }
 
+/* Check if AFMV option is provided in command */
+
+static bool afmv_enabled() {
```

Comment:
Try and match the same function declaration conventions as the others in GCC.

Ie. static then return type on the first line followed by the function name on the next line.
Make sure you explicitly put void within the parameters of the function as well.